gqlgen.yml

```diff
@@ -81,15 +81,14 @@ models:
       - github.com/99designs/gqlgen/graphql.Int64
       - github.com/99designs/gqlgen/graphql.Int32
   Int:
-    model:
-      - github.com/99designs/gqlgen/graphql.Int
-      - github.com/99designs/gqlgen/graphql.Int64
-      - github.com/99designs/gqlgen/graphql.Int32
+    model: github.com/DIMO-Network/identity-api/graph/types.Int
```

Comment:
I think it's dangerous to redefine `Int` to be a string in the output—is that what this will do? I would rather just output a number.